widget_driver_annotation/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+# CHANGELOG
+
```

Comment:
I am not sure about this format...
So yeah, I also prefer it your way.
But all the google-official-flutter repos use a CHANGELOG format where they only have this content:

```md
## {major.minor.patch}

* what changed
```

So I am not sure.. should we follow the markdown best practices.. or do we follow the google-flutter-approach?
Maybe best to follow the google-way?

Reply:
I just added the title CHANGELOG. We can follow that convention below in the file itself?

Reply:
Maybe I was not explaining my comment correctly..

The offical format of MD-files is to have a title.. Just like you did it.. So that is great.
So you added `# CHANGELOG`

**BUT** it seems that the convention for dart packages is to not have this title. Just directly start with the verions.
If you look at all google dart packages, they all skipped the title-part.. and directly jump into the versions..

I guess a reason for this is due to the design of the pub.dev webpage.. Cause if you go to a package and click on the "changelog" button then you see this:
<img width="895" alt="Screenshot 2022-12-13 at 20 09 50" src="https://user-images.githubusercontent.com/59053438/207424012-1346166d-5597-4853-b5a9-fd4093330632.png">

And there I guess they dont want to have "CHANGELOG" mentioned again.. which it would be when you define a title with that word..


But anyhow. we can always change that in the future...
So lets go use you change as-is for now.

Reply:
Ahh, I got it now. I'll change it then.